committing.rst

```diff
@@ -258,6 +258,35 @@ Developers can apply labels to GitHub pull requests).
 
 .. _cherry_picker.py: https://github.com/python/core-workflow/tree/master/cherry_picker
 
+.. _git_worktrees:
+
+Speeding up development workflow using git worktrees
+----------------------------------------------------
+It's often useful to have more than one checkout of the codebase (for testing on one version + 
+developing on another concurrently). For example::
+
+    # create a new clone of upstream/3.6 at ../3.6
+    $ git worktree add ../3.6 upstream/3.6   
+    Preparing ../3.6 (identifier 3.6)
+    HEAD is now at 49731ef8d2 dict: add iterkeys aliases in 3.5
+    $ cd ../3.6;  # go to the checkout
+    $ git cherry-pick 686c146b1ca1eb5c7ef058bb28772ac612a8ec1d  # add a patch and commit the result
+    [bpo-31620 c48137b107] bpo-31620: have asyncio/queues not leak memory when you've exceptions during waiting
+    Date: Fri Sep 29 01:03:19 2017 +0000
+    files changed, 25 insertions(+)
+    create mode 100644 Misc/NEWS.d/next/Library/2017-10-06-04-35-31.bpo-31620.gksLA1.rst
+    $ ./python -m test -j3  # run tests on the new patch
```

Comment:
This is *nix specific and does not work on Windows.

`> .\python  -m test -j3` will work, but as far as I know, the `.\` is unneeded as `python` in the repository directory overrides `python` on the path.  So either drop `./` or add
`> python -m test -j3` for the Windows version of the command.

Reply:
Dropping "./" on *nix won't work (will hit the python from the PATH, not from the repository).

Reply:
The documentation [about running tests](https://devguide.python.org/runtests/) specifies 

> You may need to change this command as follows throughout this section. On most Mac OS X systems, replace ./python with ./python.exe. On Windows, use python.bat. If using Python 2.7, replace test with test.regrtest.

and then use a lot of `./python`.

I we do not want to create two example blocs maybe use a paragraphe like it?